src/Analyzer/ReferenceTrimmerAnalyzer.cs

```diff
@@ -87,7 +86,7 @@ private static void DumpUsedReferences(CompilationAnalysisContext context)
         {
             if (metadataReference.Display != null)
             {
-                string assemblyName = AssemblyName.GetAssemblyName(metadataReference.Display).Name;
+                string assemblyName = new Uri(metadataReference.Display).LocalPath;
```

Comment:
Why are you using a `Uri`?

Reply:
This is a path normalization function

Reply:
`Path.GetFullPath` should do the same thing?